crates/node/src/print.rs

```diff
@@ -77,6 +79,8 @@ pub fn print(
 
 #[napi]
 pub fn print_sync(program: String, options: Buffer) -> napi::Result<TransformOutput> {
+    crate::util::init_trace_once(false, None)?;
```

Comment:
This repeated call is to preserve existing behavior to support `ENV_LOG` without breaking changes.

I hope to declare breaking changes to this eventually, by letting users explicitly opt in by calling trace registration.